### DISCUSSION TITLE
Fehler 74 und 47

### DISCUSSION THREAD

**Discussion by Marc9485** [tbnobody/OpenDTU#984] [Q&A]
Erstmal, vielen Dank für openDTU😉Hab da k/ein Problem?!?
Habe letzte Woche Donnerstag einen Hoymiles HM1500 mit 4x 430W-Modulen in Betrieb genommen. Alles ohne Probleme. Dienstag hab ich einen weiteren HM1500 mit 4x 430W-Modulen noch zusätzlich aufs Dach gepackt. Die WR's hab ich in Reihe verbunden (WR2/Out - WR1/In | WR1/Out - UV Garage). Ein paar Stunden nach Inbetriebnahme hab ich dann in der Ereignisanzeige von WR1 mehrere Fehler gehabt. Alle in der selben Sekunde. Over Voltage, under Voltage, over frequency, under frequency, usw... + Fehler 74(Unknown). Einen Tag später zeigt WR1 dann Fehler 47(Unknown) an. Ich weiß das noch nichts über die Fehler bekannt ist...
Ich finds halt nur komisch, dass der WR1 (solo) über 5 Tage ohne einen Fehler funktioniert und mit einem weiteren WR verbunden fangen jetzt die Meldungen an...
Ich könnte für WR2 auch grad noch  ein extra Kabel legen und dann in der UV auf L2 auflegen und schauen ob dann wieder alles i.O. ist. 
Was meint ihr?
Danke

**Comment by DPO99** [comment]
Also den Fehler 47 kenne ich auch immer in solchen Konstalltionen, wenn Grid weg war. Diese ganzen Spannungs und Frequenz Fehler lassen ja auch darauf hinweisen, dass mit dem Grid etwas nicht stimmt (weg z.B.). Könnte aber natürlich auch an einer "schlechten" Elektrik liegen. Ich gehe davon aus, dass du dich mit den physikalischen Themen (Kabellänge, Querschnitt, Leistung, etc.) auseindergesetzt hast.

Ich würde das mal beobachten, ob diese Fehler wieder auftreten, wenn du einen Tag mal nichts am WR gemacht hast. Starte die WR auch gerne mal über die DTU neu (wenn Fehler drin sind), dann kriegt man sofort und einfach mit, wenn was neues kommt.
